src/devices/Tlc1543/Tlc1543.cs

```diff
@@ -23,9 +22,11 @@ public class Tlc1543 : IDisposable
         private readonly int _endOfConversion;
         private readonly bool _shouldDispose;
         private readonly TimeSpan _conversionTime = new TimeSpan(210);
-        private GpioController _controller;
+        private GpioController? _controller;

```

Comment:
controller is never null in the lifetime so it should not be nullable (state after Dispose is not considered lifetime, you should just assign `_controller = null!;` in the Dispose method